### ISSUE TITLE
Resource 'aquasec_enforcer_groups' is not setting parameter 'host_network_protection'

### ISSUE THREAD

**Issue by violiveira** [aquasecurity/terraform-provider-aquasec#123]
Hi there,

The resource `aquasec_enforcer_groups` is not setting all parameters properly.
If one sets the following parameters for a enforcer group of type 'agent':

```
  host_assurance = true
  host_protection = true
  host_network_protection = true
  host_user_protection = false
  antivirus_protection = false
```

and performs `terraform apply`, the command succeed, but the `host_network_protection` parameter is not set correctly. 

The GUI would show the following values:

<img width="647" alt="image" src="https://user-images.githubusercontent.com/13354550/168303510-cc8369df-bcee-4f82-b9c9-3ba38b74f1bd.png">

Another evidence that something is wrong is that whenever one runs `terraform plan`, even without doing any changes to the code, it will always show the following pending changes:

```
Terraform will perform the following actions:

  # aquasec_enforcer_groups.group-name will be updated in-place
  ~ resource "aquasec_enforcer_groups" "group-name" {
      ~ container_activity_protection                  = false -> true
      ~ host_network_protection                        = false -> true
      ~ host_protection                                = false -> true
        id                                             = "..."
      + logical_name                                   = "..."
        # (46 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```
`terraform plan` always shows pending changes because the provider is simply not able to update some of the fields according to the code, therefore there is always a discrepancy between the terraform code and what comes from the Aqua API.

### Terraform Version
```
% terraform -v
Terraform v1.1.5
on darwin_arm64
+ provider registry.terraform.io/aquasecurity/aquasec v0.8.9
```

### Affected Resource(s)
Please list the resources as a list, for example:
- aquasec_enforcer_groups

**Comment by yossig-aquasec** [comment]
@violiveira can you please update to version 0.8.10 and check again please?.

**Comment by violiveira** [comment]
Hi @yossig-aquasec, thanks for checking this out. I have done more testing and I still face issues when I try to set host_network_protection from false to true:

`~ host_network_protection                        = false -> true`

If I apply the configuration below:

```
  host_assurance = true
  host_protection = true
  host_network_protection = true
  host_user_protection = false
  antivirus_protection = false
```
I am still ending up with this:

<img width="775" alt="image" src="https://user-images.githubusercontent.com/13354550/169531460-8757dc66-196d-440c-91c1-a3a58876a122.png">

However, I have noticed that the opposite setting works. If I set `host_network_protection` from `true` to `false` via terraform, it works. I hope this helps you to better isolate the issue.
